shack.json

```diff
@@ -1,22 +1,34 @@
 {
-  "$schema": "https://unpkg.com/tachometer/lib/config.schema.json",
+  "$schema": "https://raw.githubusercontent.com/Polymer/tachometer/master/config.schema.json",
+  "timeout": 0,
+  "sampleSize": 100,
   "benchmarks": [
     {
```

Comment:
why not use expand for browser and measurement?

Reply:
I think expand is bad because it's unintuitive. I wanted to share this repo with someone and I thought it could confuse them (because it's confused a lot of people in my experience). I think we should just have "browsers" at the top level to support the main use case for expand, which is "run all these benchmarks in this browser".